test/support/validate.js

```diff
@@ -48,7 +48,7 @@ function runValidation(source: string, expectedOutput: ?any) {
   let coffeeOutput = runCodeAndExtract(coffeeES5);
   let decaffeinateOutput = runCodeAndExtract(decaffeinateES5);
   try {
-    strictEqual(decaffeinateOutput, coffeeOutput);
```

Comment:
Previously, all usages of `validate` expected the value of `o` at the end to be a primitive, so an `===` check was good enough in the assertion. I added the first case where the expected result is an array, so we need to do a deeper check of the contents.